scene/gui/color_picker.cpp

```diff
@@ -675,22 +675,26 @@ void ColorPicker::_screen_input(const Ref<InputEvent> &p_event) {
 		screen->hide();
 	}
 
-	Ref<InputEventMouseMotion> mev = p_event;
-	if (mev.is_valid()) {
+	mouse_not_moving_pos = p_event;
+	mouse_not_moving_timer->start();
```

Comment:
I'd move the event `is_valid()` check here. There's no point in starting the timer if the event is invalid. Also with the current code, `get_global_position()` might be called on invalid event.